packages/ember-auto-import/ts/webpack.ts

```diff
@@ -391,6 +424,7 @@ export default class WebpackBundler extends Plugin implements Bundler {
     packageName: string;
     packageRoot: string;
   }): void {
+    // I guess this is "part 2" of what stagingDir is being used for. Not yet sure how to replace this part...
```

Comment:
Yes. The files like `./__ember_auto_import__/${bundle}.js` contain imports, and webpack will try to resolve those imported packages relative to that path. It needs to find the right ones. In the stagingDir we did that by making symlinks.

You could instead use resolve.alias or resolve.fallback. See https://webpack.js.org/configuration/resolve/ This is probably a small amount of code to write, but it could have confusing effects on people who were already doing some aliasing.

The alternative I mentioned in https://github.com/ef4/ember-auto-import/pull/479#issuecomment-1003767077 is to  actually locate the virtual modules such that they appear to be *inside* the *real* package that did the importing, so that the existing dependency resolution would work. This would be very nice because it means there's no confusion about aliases and we'd be closer to how a "normal" webpack build would work (one where webpack looks at the actual app source, instead of ember-auto-import's summary of the app source). This might need wider code changes though, so that you could know the needed imports on a per-package (meaning the app and any addons that are using ember-auto-import) basis.